ui-v2/tests/acceptance/steps/components/acl-filter-steps.js

```diff
@@ -0,0 +1,10 @@
+import steps from '../steps';
+
+// step definitions that are shared between features should be moved to the
+// tests/acceptance/steps/steps.js file
+
+export default function(assert) {
+  return steps(assert).then('I should find a file', function() {
+    assert.ok(true, this.step);
+  });
+}

```

Comment:
One of the things I don't like about the ember add-on I'm using for this is that you are forced to have an equivalent `steps` file for every `feature` file, which you generally don't need. I'd rather the blueprint didn't create them, and the addon didn't require them. I don't think it would be that hard to tweak it so it didn't need them, but that's right down the bottom of my list at the min, and they aren't currently doing any harm, they are never used.